app/views/includes/_header.html

```diff
@@ -1,32 +1,55 @@
         </svg>
-        <span class="govuk-header__product-name">Services list</span>
       </a>
     </div>
   </div>
 </header>
 
-<nav class="x-govuk-primary-navigation" aria-labelledby="primary-navigation-heading">
+<section class="govuk-service-navigation" aria-label="Service information" data-module="govuk-service-navigation">
   <div class="govuk-width-container">
-    <h2 class="govuk-visually-hidden" id="primary-navigation-heading">Menu</h2>
-    <ul class="x-govuk-primary-navigation__list">
-      <li class="x-govuk-primary-navigation__item{{ ' x-govuk-primary-navigation__item--current' if section == "home" }}">
-        <a class="x-govuk-primary-navigation__link" href="/"{{ ' aria-current="page"' if section == "home" }}>Home</a>
-      </li>
-      <li class="x-govuk-primary-navigation__item{{ ' x-govuk-primary-navigation__item--current' if section == "organisation" }}">
-        <a class="x-govuk-primary-navigation__link" href="/organisation"{{ ' aria-current="page"' if section == "organisation" }}>By organisation</a>
-      </li>
-      <li class="x-govuk-primary-navigation__item{{ ' x-govuk-primary-navigation__item--current' if section == "contribute" }}">
-        <a class="x-govuk-primary-navigation__link" href="/contribute"{{ ' aria-current="page"' if section == "contribute" }}>Contribute</a>
-      </li>
-    </ul>
+    <div class="govuk-service-navigation__container">
+      <span class="govuk-service-navigation__service-name">
+        <a href="/" class="govuk-service-navigation__link">Services List</a>
```

Comment:
Lowercase 'l' I think?

```suggestion
        <a href="/" class="govuk-service-navigation__link">Services list</a>
```

Reply:
I sort of thing of products being ‘Title Case’ (i.e. GOV.UK Forms, GOV.UK Design System) and services being ‘Sentence case’ (i.e. Apply for a thing).

I’ve updated the PR to keep the service name in the header, but capitalised it according to these ‘rules’ (which are also followed by other X-GOVUK sites).